tests/test_wave_01.py

```diff
@@ -183,12 +183,11 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
-    # *******************************************************************************************
-    # ****** Add assertions here to test that the correct movie was added to "watched" **********
-    # *******************************************************************************************
+    assert movie_to_watch in updated_data["watched"]
```

Comment:
Great use of `in` to check if `movie_to_watch` is where we expect.

We know this movie is where it belongs, but the assertion doesn't guarantee that the other movies are where we expect them. What other assertions could we add to ensure each movie is in the list we expect?